ddtrace/contrib/pytest/plugin.py

```diff
@@ -806,7 +806,14 @@ def pytest_ddtrace_get_item_suite_name(item):
     if test_module_path:
         if not pytest_module_item.nodeid.startswith(test_module_path):
             log.warning("Suite path is not under module path: '%s' '%s'", pytest_module_item.nodeid, test_module_path)
-        suite_path = os.path.relpath(pytest_module_item.nodeid, start=test_module_path)
+        try:
+            suite_path = os.path.relpath(pytest_module_item.nodeid, start=test_module_path)
+        except TypeError:
```

Comment:
Shouldn't this be
```suggestion
        except ValueError:
```
?

Reply:
Tempted to commit this based on https://docs.python.org/3/library/os.path.html .